src/inmanta/agent/forking_executor.py

```diff
@@ -542,13 +543,13 @@ def mp_worker_entrypoint(
     logging.captureWarnings(True)
 
     # Set up our own logger
-    logger = logging.getLogger(f"agent.executor.{name}")
+    logger = logging.getLogger(f"inmanta.executor.{name}")
```

Comment:
I still have some gripes about reusing the same string `"inmanta.executor"` all over this code. I think we should have it in a const